src/Documentation/sidebar.json

```diff
@@ -24,14 +24,29 @@
       {
         "label": "Get Older Files",
         "slug": "older-versions"
-      },
-      {
-        "label": "Example: Versioning",
-        "slug": "example-versioning"
-      },
+      }
+    ]
+  },
+  {
+    "slug": "tutorials",
+    "source": "tutorials/index.md",
+    "children": [
+      "basics",
+      "interactive",
+      "versioning",
```

Comment:
same here - should we come up with a bit more meaningful names? Data and Model Versioning?